modules/announcements/module.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Elementor\Modules\Announcements;
+
+use Elementor\Core\Base\App as BaseApp;
+use Elementor\Modules\Announcements\Classes\Announcement;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseApp {
+
+	/**
+	 * @return bool
+	 */
+	public static function is_active(): bool {
+		return is_admin();
```

Comment:
If the announcements should be rendered only at the Editor, you can also add something like
`Plugin::elementor()->editor->is_edit_mode()` to the conditions.